### DISCUSSION TITLE
There was an unexpected error while talking to the internal Vercel API.

### DISCUSSION THREAD

**Discussion by ml0-1337** [vercel/community#1669] [Help] [answered]
### Summary

I usually deploy  next.js a few times a day, and this error keeps insisting since last night and my website has not been updated.

> There was an unexpected error while talking to the internal Vercel API. Please check the logs for additional help and/or try again.
--
14:02:50.241 | An error was thrown and it will end the deployment but the thrown error had no message body. Perhaps other parts of the log offer a clue.

I could not figure out what is causing this error. Please help

I get 2 error messages on vercel for error, and those two above are the only ones.

### Additional information
![スクリーンショット 2023-03-02 14 23 53](https://user-images.githubusercontent.com/122081925/222338554-0789257c-dd17-4903-b120-ce6364619174.png)

**Comment by elleryfamilia-huge** [comment]
We're running into the same problem.

**Comment by amyegan** [comment]
Please share a deployment id or URL so I can look into the cause of this error 🙏

**Reply by elleryfamilia** [reply]
> Please share a deployment id or URL so I can look into the cause of this error 🙏 

4MAepeees5UWwZ3i5yPwfYtVZUyq

**Reply by ml0-1337** [reply]
D2gPKDHFMmiefFe9BtxXdHjNfrF7

**Comment by ml0-1337** [comment, marked as answer]
The issue has been solved by reducing the SSG pages per build even though we did not reach to the build time per deployment limit([Vercel API Limits](https://vercel.com/docs/concepts/limits/overview))
